GoogleApiWrappers/src/Google.Apis.Storage.v1.IntegrationTests/README.md

```diff
@@ -0,0 +1,26 @@
+Integration test prerequisites
+====
+
+The integration tests require:
+
+- A project, identified by the `TEST_PROJECT` environment variable
+- An authenticated account with read/write access to the project
+- A number of storage buckets with canned data in
+
+The last bullet here can be set up using `setup.bat`, and deleted
+with `clean.bat`. Running `setup.bat` will automatically run
```

Comment:
General though, maybe we can write these shared scripts between windows and linux/osx as python scripts that can run xplatform? Similar to how gcloud itself is implemented?